utils/linux-install-luarocks.sh

```diff
@@ -40,7 +40,7 @@ fi
 
 make build > build.log 2>&1 || (cat build.log && exit 1)
 msg="rerun this script with 'sudo' if you failed to make install because of privilege problem."
```

Comment:
We can remove this prompt and other sudo in the ci script or docs now?

Reply:
I think yes if everything is as expected. We could safely remove this prompt if it's only for `make install`. What do you think?

Reply:
Yes. We think about the same thing.